packages/core/core/__tests__/extend.test.tsx

```
@@ -0,0 +1,143 @@
+interface ButtonProps
+  extends Omit<NovaWaveUIProps<'button'>, 'color'>,
+    ExtractVariantNonSlottedProps<typeof testButtonStyles> {
+  children: React.ReactNode;
+}
+
+const Button = React.forwardRef<HTMLButtonElement, ButtonProps>(
+  (props: ButtonProps, _ref) => {
+    const { children, className, ...rest } = props;
+    const [, variantProps] = mapPropsToVariants(
+      props,
+      testButtonStyles.variantKeys!,
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)